config/dynamicconfig.yaml

```diff
@@ -0,0 +1,37 @@
+# Use this file to override the default dynamic config value (they are specified
```

Comment:
Let's move sample config to path `config/dynamic_config/dev.yaml`

Reply:
Done